avro2py/codegen.py

```diff
@@ -418,9 +419,9 @@ def populate_namespaces(schemas: List[Record]) -> Generator[Tuple[str, ast.Modul
     def frontier_sorting_key(s):
         return s.namespace not in namespace_nodes, s.namespace
 
-    frontier = sorted(schemas, key=frontier_sorting_key)
+    frontier = deque(sorted(schemas, key=frontier_sorting_key))
```

Comment:
Observe the _original_schema in the following examples.

<details><summary>Before this code change</summary>
class AdgroupCreated(NamedTuple):
    status: "Status"
    _original_schema = (
        '{"type": "record", "name": "AdgroupCreated", "namespace":'
        ' "marketprice.messages.bid", "fields": [{"name": "status", "type": {"type":'
        ' "enum", "name": "Status", "doc": "status can be one of Active, Paused",'
        ' "symbols": ["Active", "Paused"]}}]}'
    )
</details>

<details><summary>After the code change</summary>
```
class AdgroupCreated(NamedTuple):
    status: "Status"
    _original_schema = (
        '{"type": "record", "name": "AdgroupCreated", "namespace":'
        ' "marketprice.messages.bid", "fields": [{"name": "status", "type": {"type":'
        ' "enum", "name": "Status", "doc": "status can be one of Active, Paused",'
        ' "symbols": ["Active", "Paused"]}}]}'
    )
</details>